src/client/src/router.js

```diff
@@ -11,6 +11,11 @@ export default new Router({
 	mode: 'history',
 	base: process.env.BASE_URL, 
 	routes:[
+		{
+			path: '/',
+			name: 'home',
+			component: Home
+		},
```

Comment:
why are there 2 routes to '/' with the same name and component?

Reply:
yeah it's redundant.